srcpkgs/python3-fastjsonschema/template

```diff
@@ -13,7 +13,7 @@ license="BSD-3-Clause"
 homepage="https://github.com/seznam/python-fastjsonschema"
```

Comment:
```suggestion
homepage="https://github.com/horejsek/python-fastjsonschema"
```

Reply:
Thanks for spotting this, I missed this. Fixed.